File: php_src/bourdon/kernel/app/model/commercial.php
<?php
require_once(LIB."Model.php");
class commercial extends Model
{
    protected $id;
    protected $nickName;
    protected $name;
    protected $address1;
    protected $address2;
    protected $pc;
    protected $city;

    public function __construct()
    {
        parent::__construct("commercial", "id");
        $this->id = null;
        $this->nickName = null;
        $this->name = null;
        $this->address1 = null;
        $this->address2 = null;
        $this->pc = null;
        $this->city = null;
    }

    



    /**
     * @return mixed
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param mixed $id
     *
     * @return self
     */
    public function setId($id)
    {
        $this->id = $id;

        return $this;
    }

    /**
     * @return mixed
     */
    public function getNickName()
    {
        return $this->nickName;
    }

    /**
     * @param mixed $nickName
     *
     * @return self
     */
    public function setNickName($nickName)
    {
        $this->nickName = $nickName;

        return $this;
    }

    /**
     * @return mixed
     */
    public function getName()
    {
        return $this->name;
    }

    /**
     * @param mixed $name
     *
     * @return self
     */
    public function setName($name)
    {
        $this->name = $name;

        return $this;
    }

    /**
     * @return mixed
     */
    public function getAddress1()
    {
        return $this->address1;
    }

    /**
     * @param mixed $address1
     *
     * @return self
     */
    public function setAddress1($address1)
    {
        $this->address1 = $address1;

        return $this;
    }

    /**
     * @return mixed
     */
    public function getAddress2()
    {
        return $this->address2;
    }

    /**
     * @param mixed $address2
     *
     * @return self
     */
    public function setAddress2($address2)
    {
        $this->address2 = $address2;

        return $this;
    }

    /**
     * @return mixed
     */
    public function getPc()
    {
        return $this->pc;
    }

    /**
     * @param mixed $pc
     *
     * @return self
     */
    public function setPc($pc)
    {
        $this->pc = $pc;

        return $this;
    }

    /**
     * @return mixed
     */
    public function getCity()
    {
        return $this->city;
    }

    /**
     * @param mixed $city
     *
     * @return self
     */
    public function setCity($city)
    {
        $this->city = $city;

        return $this;
    }
}

File: src/app/src/main/java/com/example/issac/myapplication/MODEL/Commercial.java
package com.example.issac.myapplication.MODEL;

import org.json.JSONException;
import org.json.JSONObject;

import java.util.ArrayList;

public class Commercial extends Model {
    protected int id;
    private String name;
    private String nickName;
    private String address1;
    private String address2;
    private String pc;
    private String city;

    public Commercial() {
        super("commercial");            // on déclare le nom de la table
    }

    /*
    07/05
    Matthieu (basé sur Boutte)
    Permet d'assigner des attributs d'un objet avec un élément JSON
     */

    public void putInObj(JSONObject json) {
        try {
            this.name = json.getString("name");                 // ici on assigne la totalité des attributs avec le résultat en JSON
            this.nickName = json.getString("nickName");
            this.address1 = json.getString("address1");
            this.address2 = json.getString("address2");
            this.pc = json.getString("pc");
            this.city = json.getString("city");
        } catch (JSONException e) {
            e.printStackTrace();
        }

    }

    public ArrayList<Commercial> JSONArrayToCommercials(ArrayList<JSONObject> table) {
        ArrayList<Commercial> lesCommercials = new ArrayList<Commercial>();
        int i;
        for(i=0;i<table.size();i++) {
            Commercial leCommercial = new Commercial();
            leCommercial.putInObj(table.get(i));
            lesCommercials.add(leCommercial);
        }
        return lesCommercials;
    }

    @Override
    public int getId() {
        return id;
    }

    @Override
    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getNickName() {
        return nickName;
    }

    public void setNickName(String nickName) {
        this.nickName = nickName;
    }

    public String getAddress1() {
        return address1;
    }

    public void setAddress1(String address1) {
        this.address1 = address1;
    }

    public String getAddress2() {
        return address2;
    }

    public void setAddress2(String address2) {
        this.address2 = address2;
    }

    public String getPc() {
        return pc;
    }

    public void setPc(String pc) {
        this.pc = pc;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }
}
